hydra-node/json-schemas/api.yaml

```
@@ -1245,6 +1243,18 @@ components:
             tag:
               type: string
               enum: ["FailedToDraftTxNotInitializing"]
+        - title: FailedToDraftTxWalletUtxoDetected
```

```suggestion
        - title: SpendingNodeUtxoForbidden
```

The user is already in a context where they try to draft a tx so I would remove _FailedToDraft_ from the name.

Since we detected Utxo from the node's wallet, I would make it clear that it is forbidden to try to spend this UTXO.

---

No strong opinion about the name, but much more important: this schema is unused!

Must: mention this (and other errors) as a possible response on the `/commit` endpoint.